.github/workflows/manual_gcs_release_test.yml

```diff
@@ -15,3 +15,5 @@ jobs:
         uses: google-github-actions/setup-gcloud@v2
       - name: Test Gsutil
         run: gsutil ls gs://bdot-release
+      - run: gsutil cp LICENSE gs://bdot-release/LICENSE
+      - run: gsutil rm gs://bdot-release/LICENSE
```

Comment:
Because this is a public bucket, we need to perform a copy and rm to ensure we have the correct access. These are safe operations in the root of the bucket.